apps/hashdotai/guide/10_flows/01_triggers.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Trigger Steps
+description: Automating when flows are run
+metaTitle: Triggers - HASH
+metaDescription: Triggers can be used to specify when a flow should run: on a predefined schedule, or in response to an external event
```

Comment:
Need to quote anything in frontmatter with a `:` in it otherwise it won't be parsed (and the doc won't show)
```suggestion
metaDescription: "Triggers can be used to specify when a flow should run: on a predefined schedule, or in response to an external event"
```